city_scrapers/spiders/il_finance_authority.py

```diff
@@ -0,0 +1,171 @@
+        title = meeting_dict["title"]
+        date = meeting_dict["date"]
+        time = meeting_dict["time"]
+        location = meeting_dict["location"]
+
+        meeting = Meeting(
+            title=title,
+            description="",
+            classification=self._parse_classification(title),
+            start=self._meeting_datetime(date, time),
+            end=None,
+            all_day=False,
+            time_notes="",
+            location=location,
+            links=self._parse_links(response.url, title),
+            source=self._parse_source(response),
+        )
+        meeting["status"] = self._get_status(meeting)
+        meeting["id"] = self._get_id(meeting)
+        yield meeting
+
+    def _meeting_datetime(self, date, time):
+        meeting_start = date + " " + time
+        meeting_start = meeting_start.replace(", ", ",").strip()
+        return datetime.strptime(meeting_start, "%b %d,%Y %I:%M %p")
+
+    def _get_pdf_link(self, item):
+        pdf_tag = item.css("td:nth-child(4) > a")
+        if not (pdf_tag):
+            return None
+        pdf_link = pdf_tag[0].attrib["href"]
+        return pdf_link
+
+    def _parse_title(self, item):
+        """Parse or generate meeting title."""
+        try:
+            title = item.css("td:nth-child(3)::text").extract_first()
+            return title
+        except TypeError:
+            return ""
+
+    def _parse_classification(self, title):
+        """Parse or generate classification from allowed options."""
+        if "Comm" in title:
+            return COMMITTEE
+        if "Board" in title:
+            return BOARD
+        return COMMISSION
+
+    def _parse_location(self, pdf_content):
+        try:
+            """Parse or generate location."""
+            address_match = re.search(
+                r"(?:in\s*the|at\s*the) .*(\. | \d{5})", pdf_content
+            )
+            address = address_match.group(0)
+            name = re.findall(r"(?:in\s*the|at\s*the).*?,", pdf_content)[0]
+        except Exception:
+            address = "Address Not Found"
+            name = "Name Not Found"
+        return {"address": address, "name": name}
+
+    def _parse_date(self, item):
+        """Parse start datetime as a naive datetime object."""
+        try:
+            date_str = item.css("td:nth-child(2)::text").extract_first()
+            return date_str
+        except TypeError:
+            return ""
+
+    def _parse_start(self, pdf_content):
+        try:
+            time = re.findall(r"\d{1,2}:\d{2}\s?(?:A.M.|P.M.|PM|AM)", pdf_content)[0]
+            return time
+        except Exception:
+            return "12:00 AM"
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_all_day(self, item):
+        """Parse or generate all-day status. Defaults to False."""
+        return False
+
+    def _parse_links(self, link, title):
```

Comment:
It looks like there are multiple links here, so ideally we would want to pull the notice, minutes, and any other information as links even if we aren't parsing them.

Reply:
Are you talking about the the other pdf links on the website? Could you elaborate more? 
Is there an example you could provide? Thanks

Reply:
Sure, I'm seeing up to 4 PDF links in each row for the Agenda, Board Book, Minutes, and Voting Record. Ideally we'll want to include all of those in the `links` list